lab_5_scrapper/scrapper.py

```diff
@@ -90,25 +196,41 @@ def __init__(self, config: Config) -> None:
         """
         Initializes an instance of the Crawler class
         """
-        pass
+        self._config = config
+        self._seed_urls = config.get_seed_urls()
+        self.urls = []
 
     def _extract_url(self, article_bs: BeautifulSoup) -> str:
         """
         Finds and retrieves URL from HTML
         """
-        pass
+        url = article_bs.get('href')
+        if isinstance(url, str) and url.startswith('https://kazanfirst.ru/news/') and len(url) > 27:
```

Comment:
it cannot be a number, it is either None or str

Reply:
fixed it